### DISCUSSION TITLE
What is the use case for the new Price Check all same-day flights?

### DISCUSSION THREAD

**Discussion by natecodes** [jdholtz/auto-southwest-check-in#310] [General]
### Check Fares Values
- `false` or `"no"`: Do not check for lower fares
- `true` or `"same_flight"`: Check for lower fares on the same flight
- `"same_day_nonstop"`: Check for lower fares on all nonstop flights on the same day as the flight
- `"same_day"`: Check for lower fares on all flights on the same day as the flight

Is this use case for same_day checking for someone that has A-list who can same-day no-fee change? I don't get what the point of this is

**Comment by jdholtz** [comment]
When checking fares on your flight in Southwest, it brings up all similar flights (same day as the departure and same departing/arrival flight). 

The `same_day` option will check if any of these flights during the entire day are lower than your current flight. This could be used by someone if they want to check if there is a cheaper flight at all during the day rather than just on the exact one they booked.
